src/NUnitFramework/framework/Internal/Filters/NotFilter.cs

```diff
@@ -57,15 +57,15 @@ public NotFilter( TestFilter baseFilter)
         /// <returns>True if the test passes the filter, otherwise false</returns>
         public override bool Pass(ITest test)
         {
-            return !BaseFilter.Pass(test);
-        }
+			return !BaseFilter.Match (test) && !BaseFilter.MatchParent (test);
+		}
 
-        /// <summary>
-        /// Check whether the filter matches a test
-        /// </summary>
-        /// <param name="test">The test to be matched</param>
-        /// <returns>True if it matches, otherwise false</returns>
-        public override bool Match( ITest test )
+		/// <summary>
+		/// Check whether the filter matches a test
+		/// </summary>
+		/// <param name="test">The test to be matched</param>
+		/// <returns>True if it matches, otherwise false</returns>
+		public override bool Match( ITest test )
```

Comment:
I believe the file you saw is for use with resharper. In any case, I know that I had to set my VS settings to match our standards in order to get it to work. I don't believe there's a per-solution way to do it. @rprouse you know more about this than I do... is that right?

In any case, if you can't fix it easily, let's not worry about it right now.